data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -375,6 +375,26 @@ spec:
                           required:
                           - DiskSizeGB
                           type: object
+                        osImage:
+                          description: OSImage defines a custom image for instance.
+                          properties:
+                            name:
+                              description: Name defines the name of the image.
+                              type: string
+                            project:
+                              description: Project defines the name of the project
+                                containing the image.
+                              type: string
+                          required:
+                          - name
+                          - project
+                          type: object
+                        tags:
```

Comment:
@barbacbd Looks like you are introducing user specified tags here and on lines 1047 and 2238. Is that intentional? We released that enhancement in 4.12. https://issues.redhat.com/browse/CORS-2209